src/index.js

```diff
@@ -1 +1,56 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+// {
+//   "result": "Game with ID: JQg6oO2j3TrBWpEacMxL added."
+// }
+
```

Comment:
- [ ] Kindly make sure to remove all commented code to make your project cleaner and professional :recycle: